core/common/src/main/java/alluxio/Configuration.java

```diff
@@ -444,6 +444,9 @@ public static void loadClusterDefault(InetSocketAddress address) throws AlluxioS
         throw new UnavailableException(String.format(
             "Failed to handshake with master %s to load cluster default configuration values",
             address), e);
+      } catch (UnauthenticatedException e) {
+        throw new RuntimeException(
+            "Received authentication exception with authentication disabled.", e);
```

Comment:
Here I mention we should not get into this state because I disabled authentication during channel building by calling disableAuthentication(). Such that, channel won't even bother authenticating with the server and calls RPCs directly.

Now, should we allow such calls is a question that we need to answer. 
Earlier, with thrift, we allowed that. That's how this code was working before, likewise, without authentication. With the new security infra we have 2 options:
* Allow unauthenticated calls with no user information for services that requires authentication. The server call run under no user context.
* Explicitly allow un-authenticated calls to certain services, eg. MetaService,. This is viable for services that we know there is no user context.